### PR TITLE
GitHub Pull Request Template

### LINKED ISSUE #593: Create an template for PRs

### PR DESCRIPTION
**Pull request by acheshkov**
closes #593 
This is a template for PRs. To unify. Please review and make suggestions.

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Please specify the following:
+
```
Comment (lyriccoder) [inline .github/pull_request_template.md:2]: add tests

Comment (acheshkov) [reply .github/pull_request_template.md:2]: @lyriccoder I've added

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+1. A link to the original issue using `#` selector.
```
Comment (aravij) [inline .github/pull_request_template.md:3]: Where is this link goes? Title, description or I can just link PR with issue i github?

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+2. A description/notes about this pull request.
+3. If you fix a bug don't forget about test case.
```
Comment (aravij) [inline .github/pull_request_template.md:5]: New functionality needs tests as well.

Comment (acheshkov) [reply .github/pull_request_template.md:5]: added

File: .github/pull_request_template.md (hunk 2 above)
Comment (acheshkov) [reply .github/pull_request_template.md]: I've added, please check. I'm not sure I understood you correctly

Comment (aravij) [reply .github/pull_request_template.md]: I mean does this link have to be in PR title or description?

**Comment by acheshkov** [comment]
@rultor merge

**Pull request merged**